### PR TITLE
Adding search criteria with unitests

### PR DESCRIPTION
**Pull request by Sl3v7n**

### CODE REVIEW

File: src/com/jalasoft/search_files/search/search_criteria.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+This module will contain the main criteria for a specific search
+"""
+from src.com.jalasoft.search_files.utils.logging import LOGGER as LOGGER
+
+class Search_Criteria(object):
+    """
+     This class contains the methods that will be used to search.
+    """
+
+    def __init__(self, criteria=3, path='/', advanced_search=False):
```
Comment (mauricioZelaya) [inline src/com/jalasoft/search_files/search/search_criteria.py]: please add a get advance search I will use it in search class

File: src/com/jalasoft/search_files/search/search_criteria.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        :param path: A path where the search will be made.                   type: string.
+        :param advanced_search: Determine the type of search.                type: boolean.
+        """
+
+
+        self.basic_search = {
+            "criteria": criteria,
+            "path": path,
+        }
+
```
Comment (mauricioZelaya) [inline src/com/jalasoft/search_files/search/search_criteria.py]: all the attribute names should start with underscore

**Review by mauricioZelaya — CHANGES_REQUESTED** [review]

**Comment by mauricioZelaya** [comment]
please review the comments

File: src/com/jalasoft/search_files/search/search_criteria.py (hunk 1 above)
Comment (Sl3v7n) [reply src/com/jalasoft/search_files/search/search_criteria.py]: you mean flag,.. advance search is already there,.. I'm adding right now a method to get that flag

**Review by mauricioZelaya — APPROVED** [review]

**Pull request merged**